css/styles.css

```diff
@@ -0,0 +1,126 @@
+* { 
+  box-sizing: border-box; 
+}  
+
+h1{
+  color: #fdf65e;
+  background-color: #000;
+  padding: 8px 8px 8px 8px;
+  margin:0px;
+  font-size:40px;
+}
+
+nav{
+  
+  background-color: #000;
+  padding: 8px 8px 8px 8px;
+  /*float: right;*/
+  display:block;
+  text-align: right;
+  border-bottom: #fdf65e solid 4px ;
+}
+a{
+  color:#fff;
+  font-size:30px;
+  text-decoration: none;
+  margin-right: 5px;
+}
+.speciallink{
+  font-style: bold;
+  font-family: impact;
+}
+
+h2{
+  text-align: center;
+  font-style: italic;
+  font-size:30px;
+  background-color: #c6c6c6;
+  margin:0px;
+  padding: 12px 12px 12px 12px;
+}
+
+.taste{
+color:#999999;
+text-decoration: line-through;
+}
+
+.are{
+border-bottom: solid #fdf65e;
+}
+
+.background{
+  position:center center fixed;
```

Comment:
@akallbold This should be `background-position` and it cannot take a value of fixed. They use `top, right bottom, left, %, em, and px` interchangeably. 🔨

Reply:
🔨 should be fixed!